src/_layouts/base.html

```diff
@@ -159,27 +159,20 @@
     </div>
 </div>
 
-<div class="sliding-nav">
-    {% block primary_nav %}
-        {% include 'templates/nav/primary-nav.html' %}
-    {% endblock primary_nav %}
-
-    <div class="sliding-nav_page">
-    {% block header %}
-        {% include 'templates/header.html' %}
-    {% endblock header %}
-
-    <!-- PRIMARY CONTENT -->
-    {% block content %}
-        This will be replaced in templates that extend this template and override "content".
-    {% endblock content %}
-     <!-- /PRIMARY CONTENT -->
-
-    {% block footer %}
-        {% include 'templates/footer.html' %}
-    {% endblock footer %}
-    </div><!-- END .sliding-nav_page -->
-</div><!-- END .sliding-nav -->
+{% block header %}
+    {% include 'templates/header.html' %}
+{% endblock header %}
+
+<!-- PRIMARY CONTENT -->
+{% block content %}
+    This will be replaced in templates that extend this template
```

Comment:
This comment could be in a jinja comment. Or is the idea that if this was incorrectly used directly this text would show up and shame the developer? Even in that scenario I don't think we'd want to have this possibly show up on production.